docs/guidelines/index.md

```diff
@@ -0,0 +1,176 @@
+
+In your proposal, be sure to disclose all known commitments that may overlap with any of the program phases(community bonding, coding periods, evaluation periods, etc.):
+
+- Disclose your vacations periods, part-time or full-time job, school, classes, tests, exams, periods of non-availability, etc.
+- Unexpected events: we understand there can be unexpected events in life, and those cannot be planned. Please inform us as soon as possible if you need time away from the program. You can use private messaging for sensitive information.
+
+## GSoC Contributor Selection Process
+
+Once the application period is over, administrators and mentors make a decision on which proposal to accept based on the proposal submitted to the Google Summer of Code website.
+
+## Congratulations, you have been selected…​ or not.
+
+We thank all GSoC contributors who reach out to us during the application period. If you have not been selected, [read this](https://google.github.io/gsocguides/student/being-turned-down), there could be many reasons, and some are even outside of our control. Do not feel bad, we encourage you to stay with the community and apply again next year.
+
+If you have been selected, we extend our warmest congratulations! The community bonding period starts within two days after the official announcement, so get ready to dive into the exciting world of open-source development with FOSSology.
+
+## Community Bonding
+
+As soon as the GSoC contributors are accepted, the community bonding period starts. During this period, contributors are not expected to be coding immediately. Instead, they are expected to prepare to code.
+A thriving community bonding period leads to successful coding, so make the most of it!
+
+Use the community bonding to:
+
+- Define the communication channels with your mentors.
+- Schedule weekly meetings with your mentors.
+- Continue to discuss and plan the project with the community and the mentors.
+- Set up your computer and development environment to work on the project (see [Useful links](#useful-links)).
+
+## Coding Periods
+
+During the coding periods, GSoC contributors are expected to:
+
+- Work on the GSoC project.
+- Follow the [Code Style Best Practices](https://github.com/fossology/fossology/wiki/Coding-Style).
+- Write a short summary of the work done each week, published to FOSSolgy GSoC tracket. It is also advisable to publish in a personal blog as well.
+    - ``It’s okay to say things like <this> and <that> were challenging because of <reason>.``
+- Say something when you are stuck, lost in the code, confused about the objectives, etc.
+- Notify in advance when taking time off and make use of weekends to rest and avoid burnout.
+- Be present online and proactively reach out to the community if required.
```

Comment:
Please give hints to document weekly progress onto place like: https://fossology.github.io/gsoc/docs/2022/reuse/updates/2022-06-13